castle/cms/upgrades.zcml

```diff
@@ -464,4 +464,21 @@
     handler=".upgrades.upgrade_2_6_31"
     profile="castle.cms:default"
     />
+
+  <genericsetup:registerProfile
+    name="2_6_32"
+    title="CastleCMS upgrade to 2.6.32 profile"
+    directory="profiles/2_6_32"
+    description=""
```

Comment:
probably a good idea to throw in a short description like `2632 adds script that does something with the audit log`